example/properties_directory/structural/covalent_bonds.json

```diff
@@ -0,0 +1,36 @@
+[
+    {
+        "atomPair": [
+            {
+                "id": 1,
+                "value": "H"
+            },
+            {
+                "id": 2,
+                "value": "O"
+            }
+        ],
+        "bondType": "single",
+        "bondLength": {
+            "units": "angstrom",
+            "value": 0.99
+        }
```

Comment:
It might be best to only store the required information - such as which atom ids are connected and the bond type. Other information, i.e. length, atomic symbols, we can obtain from the Basis